Applications:added nrf54h to matter-bridge (#16418)
nrfconnect/sdk-nrf: applications/matter_bridge/doc/matter_bridge_description.rst

For information about how to upgrade the device firmware using a PC or a smartphone, see the :ref:`matter_bridge_app_dfu` section.

> At that moment DFU over BT_SMP in unavaliable on nrf54h20 board.

Reviewer: You can use the "note" format to make this information better visible on the page:
```suggestion
.. note::
    Currently the DFU over Bluetooth LE SMP and Matter OTA are not available for the nrf54h20 board target.
```

For information about how to upgrade the device firmware using a PC or a smartphone, see the :ref:`matter_bridge_app_dfu` section.

.. note::

>     Currently the DFU over Bluetooth LE SMP and Matter OTA are not available for the nrf54h20 board target.

Reviewer 2: This is not a board target, a board target is the full device name e.g. nrf54h20dk/nrf54h20/cpuapp, not nrf54h20, that is just a board

For information about how to upgrade the device firmware using a PC or a smartphone, see the :ref:`matter_bridge_app_dfu` section.

.. note::

>     Currently the DFU over Bluetooth LE SMP and Matter OTA are not available for the nrf54h20dk/nrf54h20/cpuapp board target.

Reviewer 3: ```suggestion
    Currently the DFU over Bluetooth LE SMP and Matter OTA are not available for the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
```

   west build -b nrf7002dk/nrf5340/cpuapp -- -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf"

.. parsed-literal::
   :class: highlight

>    west build -b nrf54h20dk/nrf54h20/cpuapp -- -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf"

Reviewer: We use hci_ipc here, right? Also you forgot to add the nRF7002 shield argument.

      west build -b nrf5340dk/nrf5340/cpuapp -p -- -Dmatter_bridge_SHIELD=nrf7002ek -DSB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE=y -DSB_CONFIG_DFU_MULTI_IMAGE_PACKAGE_WIFI_FW_PATCH=y -DSB_CONFIG_WIFI_NRF700X=y -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3

> You can program nRF54h20 board with nRf7002 EB,using following command:

Reviewer: Could you expand the description a little bit to look similar (or the same), as here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/matter/lock/README.html#building_and_running. I mean this part: "To use nrf54H20 DK with the nrf7002ek shield attached  ..."

      west build -b nrf5340dk/nrf5340/cpuapp -p -- -Dmatter_bridge_SHIELD=nrf7002ek -DSB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE=y -DSB_CONFIG_DFU_MULTI_IMAGE_PACKAGE_WIFI_FW_PATCH=y -DSB_CONFIG_WIFI_NRF700X=y -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3

> To use nrf54H20 DK with the nrf7002ek shield attached (2.4 GHz or 5 GHz), follow the Developing with nRF7002 EB user guide to connect all required pins and then use the following command to build the sample:

Reviewer: We need some editorial change and adding a hyperlink to the guide instead of plain text.
```suggestion
To use nrf54H20 DK with the ``nrf7002ek`` shield attached (2.4 GHz or 5 GHz), follow the :ref:`ug_nrf7002eb_nrf54h20dk_gs` user guide to connect all required pins and then use the following command to build the sample:
```

> To use nrf54H20 DK with the ``nrf7002ek`` shield attached (2.4 GHz or 5 GHz), follow the :ref:`ug_nrf7002eb_nrf54h20dk_gs` user guide to connect all required pins and then use the following command to build the sample:

Reviewer 3: ```suggestion
To use the nRF54H20 DK with the ``nrf7002ek`` shield (2.4 GHz or 5 GHz), follow the :ref:`ug_nrf7002eb_nrf54h20dk_gs` user guide to connect all required pins.
Once completed, run the following command to build the sample:
```
Reworded for clearer flow
